website/docs/r/nginx_certificate.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Nginx"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_nginx_certificate"
+description: |-
+  Manages a Nginx Certificate.
+---
+
+# azurerm_nginx_certificate
+
+Manages a Nginx Certificate.
```

Comment:
since this is related to the deployment, I'm guessing this should be:

```suggestion
  Manages a Certificate for an NGinx Deployment.
---

# azurerm_nginx_certificate

Manages a Certificate for an NGinx Deployment.
```